noir-projects/noir-protocol-circuits/crates/types/src/delayed_public_mutable/mod.nr

```diff
@@ -11,15 +11,12 @@ pub mod with_hash;
 
 pub fn compute_delayed_public_mutable_time_horizon<T, let INITIAL_DELAY: u64>(
     delayed_public_mutable_values: DelayedPublicMutableValues<T, INITIAL_DELAY>,
-    historical_timestamp: u64,
+    anchor_timestamp: u64,
```

Comment:
Used `anchor_timestamp` here because it genuinely always refers to anchor block timestamp.